compute/cloud-client/src/test/java/compute/InstanceTemplatesIT.java

```diff
@@ -185,4 +188,28 @@ public void testListInstanceTemplates() throws IOException {
     assertThat(stdOut.toString()).contains(TEMPLATE_NAME_WITH_SUBNET);
   }
 
+  @Test
+  public void testCreateInstanceBulkInsert() {
+    List<Instance> instances = new ArrayList<>();
+    try {
+      String namePattern = "i-##-" + UUID.randomUUID().toString().replace("-", "").substring(0, 5);
+      instances = CreateInstanceBulkInsert
+              .bulkInsertInstance(PROJECT_ID, DEFAULT_ZONE, TEMPLATE_NAME,
+                      3, namePattern, 3, new HashMap<>());
+      Assert.assertEquals(3, instances.size());
+      Assert.assertTrue(instances.stream().allMatch(instance -> instance.getName().contains("i-")));
```

Comment:
Wondering if this a fool proof way to assert that the instances were created with bulkInsert? Can we assert `i-##-` ?